code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -82,7 +82,7 @@
 							var/imageloc = synd_mind_1.current
 							if(istype(synd_mind_1.current.loc,/obj/mecha))
 								imageloc = synd_mind_1.current.loc
-							var/image/I = image('icons/mob/mob.dmi', loc = imageloc, icon_state = "synd")
+							var/image/I = image('icons/mob/mob.dmi', loc = imageloc, icon_state = (synd_mind_1 == ticker.mode.nukeop_leader) ? "synd-L" : "synd")
```

Comment:
Don't copy paste the nuke ops leader check around.

Reply:
I don't understand the problem with this check.

Reply:
Make a macro or something to check whether a mind is the nukeops leader. Don't copy paste it.

Reply:
The other nukeop check macros work in the same fashion (using mobs rather than minds).